src/byro/common/models/configuration.py

```diff
@@ -56,6 +63,19 @@ class Configuration(ByroConfiguration):
     registration_form = JSONField(
         null=True, blank=True,
     )
+    public_base_url = models.URLField(  # Do we want this here or in the settings.py next to SITE_URL?
```

Comment:
Hmm, yeah, this should go into settings.
When the code was a plugin, its configuration was naturally in the DB, but now that it's in core, the configuration should go into settings.py (and can be used by other code in the future).